recipe/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
     - python
     - urllib3 >=1.25
     - setuptools  # [py >= 312]
-    - typing_extensions >=4.0  # [py < 310]
+    - typing_extensions >=4.0  # [py <= 310]
```

Comment:
Posted https://github.com/jelmer/dulwich/issues/1735 to hopefully get this fixed upstream.